contracts/base/IMarketObserverDataTypes.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity 0.6.12;
+
+/**
+ * @dev Type constants for market observer extra metadata.
+ */
+abstract contract IMarketObserverDataTypes {
+  /**
+   * @dev State: Tranch token initial sale
+   */
+  uint256 constant public MODT_TRANCH_SALE = 1;
+  /**
+   * @dev State: Tranch token buyback
+   */
+  uint256 constant public MODT_TRANCH_BUYBACK = 2;
```

Comment:
@hiddentao I'm thinking if we should use a smaller data type like uint8 for these variables since they aren't large enough. Optional

Reply:
We could, though keeping everything as `uint256` just means fewer chances of bugs. We could revisit this later on if we feel compelled.